Described CU-TU allocation and packing instructions in EN+DE

metas-Kay opened the pull request:
- CU-TU Allocation
- Add packing items to price
- minor lexic and layout check

(this is about and subsequent to issue #260 )

metas-Kay committed:
Described CU-TU Allocation in EN+DE

(this is about issue #260 )

metas-Kay committed:
Described how to add packing items in EN+DE

(this is about issue #260 )

metas-Kay committed:
Minor lexic and layout correction

(this is subsequent to issue #260 )

metas-dh reviewed (commented):
Looks good to me; i'd suggest to add a short explanation or mention (just half a sentence or sth) that there is the possibility to set a price UOM different from the product's stock UOM, and why this could be useful, like in the UOM conversion docu. I feel like there is sth missing, for a user with no or little experience, between step 2 & 3, in "how to add packing items".

metas-Kay committed:
Added link to "add packing instructions to price"

(this is about issue #260 )

metas-Kay committed:
Updated DE ref add_packing-instructions_to_price

- Explained usefulness in overview
- added illustrative explanation for "attribute dependency" with pngs
(this is about issue #260 )

metas-Kay committed:
Updated EN ref add_packing-instructions_to_price

- Explained usefulness in overview
- added illustrative explanation for "attribute dependency" with pngs
(this is about issue #260 )

metas-Kay committed:
Update Packvorschrift_Preis_anhaengen.md

metas-Kay commented:
@metas-dh: Please verify changes/improvements. Thx

metas-dh commented:
thx for the changes.
@metasnw: documentation is ok.
